### ISSUE TITLE
Deploy all apps in parallel during a restart

### ISSUE THREAD

**Issue by judepereira** [storyscript/platform-engine#290]
Right now, a restart of the engine takes forever to boot up, since it deploys apps sequentially. However, we can parallelise this to deploy 100 apps concurrently.

**Comment by judepereira** [comment]
Acceptance criteria - apps are deployed in parallel on a restart.

**Comment by judepereira** [comment]
Also - timeout an app deployment if it takes more than a minute. This can be part of another issue.
